client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/test/shipping-rate.js

```diff
@@ -94,7 +95,9 @@ describe( 'ShippingRate', () => {
 		} );
 
 		it( 'renders the delivery date', () => {
-			expect( shippingRateWrapper ).to.contain( <div className="rates-step__shipping-rate-delivery-date">January 1</div> ); // eslint-disable-line
+			const expectedDate = moment( shippingRateWrapper.props().rateObject.delivery_date ).format( 'LL' ).split( ',' )[0];
```

Comment:
Is this the timezone issue? :D

Reply:
Yes! The issue was that the time provided in `createShippingRateWrapper()` (which is `2020-01-01T23:59:00.000Z`) can, in some timezones, not match the test's expectation of `January 1`. :)